apps/admin/base/kustomize.yaml

```diff
@@ -18,7 +18,7 @@ metadata:
   namespace: flux-system
 spec:
   interval: 10m
-  path: ./apps/admin/csi-secret-store-provider-v1.0.1
+  path: ./apps/admin/secrets-csi-driver-crds
```

Comment:
Will this not update all envs?

Reply:
I'm stuck here, not sure which option to take, tried using ours originally adding path to csi-sercret-store-provider-v1.0.1, then switched to use charts, the chart version now matches ours 1.0.1 rather than 0.2.0. CSI wouldn't load without adding this.